psc/Main.hs

```diff
@@ -78,6 +78,20 @@ compile (PSCMakeOptions inputGlob inputForeignGlob outputDir opts usePrefix) = d
             hPutStrLn stderr (P.prettyPrintMultipleWarnings (P.optionsVerboseErrors opts) warnings')
           exitSuccess
 
+globWarningOnMisses :: String -> [FilePath] -> IO [FilePath]
+globWarningOnMisses fileType = go ([], [])
+  where
+  go (hits, misses) [] = do
+    unless (null misses) $ do
+      let formattedMisses = tail $ init $ show misses
```

Comment:
I initially used that, but it was nastier than showing and chopping the ends off. The `concatMapM` pattern leads me to printing a warning for each glob pattern anyways, so this will be much cleaner